ui/app/AppLayouts/Wallet/popups/swap/SwapModalAdaptor.qml

```diff
@@ -76,6 +76,8 @@ QObject {
         filters: ValueFilter { roleName: "isTest"; value: root.swapStore.areTestNetworksEnabled }
     }
 
+    signal refreshTimer()
```

Comment:
```suggestion
    signal restartTimer()
```

?

Reply:
Also wondering why would the data layer care about something that's UI specific (the 15 seconds timer)?

Reply:
So the thing is we need to start refetching the proposal once we have received a valid one and hence this flow, I could for example rename this to suggestedRoutesReady instead, so it doesnt mean that it should restart the timer but a more generic signal.

Reply:
As you want, no strong opinion here 🙂